changelog/_template.rst.jinja

```diff
@@ -0,0 +1,33 @@
+.. _vp{{ versiondata.version.replace(".", "p") }}:
+
+{{ top_line }}
+{{ top_underline * top_line|length }}
+{% for section, _ in sections.items() %}
+{% set underline = underlines[0] %}
+
+{% if sections[section] %}
+{% for category, category_data in definitions.items() if category in sections[section] %}
+{{ category_data['name'] }}
+{{ underline * category_data['name']|length }}
+{% if category_data['showcontent'] %}
+{% for text, issues in sections[section][category].items() %}
+{% if '\n' not in text.strip() %}
+- {{ text }} ({{ issues|join(', ') }})
+{% else %}
+{% set first_line, rest = text.strip().split('\n', 1) %}
```

Comment:
This appears to have issues with CRLF, as it splits the line after the CR which results in the issue reference being put on the next file 🤔 
![image](https://user-images.githubusercontent.com/8530778/179012900-53881487-0932-4681-bb6e-e0a52feed0f5.png)